static/sass/_pattern_p-strip.scss

```diff
@@ -4,6 +4,35 @@
   @include jaas-p-strip-half-and-half;
   @include jaas-p-strip-muted-suru;
   @include charmhub-p-strip;
+  @include p-suru-light;
+}
+
+@mixin p-suru-light {
+  .has-suru {
+    background-color: $color-light;
+    background-image:
+      linear-gradient(
+        -48deg,
+        rgba(205, 205, 205, 0.2),
+        rgba(205, 205, 205, 0.2) 35%,
+        rgba(247, 247, 247, 0) 35%,
+        rgba(247, 247, 247, 0) 100%
+      ),
+      linear-gradient(
+        16deg,
+        rgba(247, 247, 247, 0),
+        rgba(247, 247, 247, 0) 80%,
+        rgba(229, 229, 229, 0.5) 80%,
+        rgba(229, 229, 229, 0.5) 100%
+      );
+    background-position: top right, top right;
+    background-repeat: no-repeat;
+    background-size: 25% 100%, 125% 250%;
+
+    @media screen and (max-width: $breakpoint-medium) {
```

Comment:
Why not wrap all the styles above in a `@media screen and (min-width: $breakpoint-medium)` and then you are not overriding and leaving `background-position` styling applied.